src/current/v24.3/cluster-virtualization-metric-scopes.md

```diff
@@ -1666,8 +1785,26 @@ SERVER: n/a
 - `storage.wal.failover.switch.count`
 - `storage.wal.failover.write_and_sync.latency`
 - `storage.wal.fsync.latency`
+- `storage.write-amplification`
```

Comment:
@rmloveland FYI, and the block just below

Reply:
oooh thanks @mdlinville - these `storeliveness` things are related to https://github.com/cockroachdb/docs/pull/18989